### DISCUSSION TITLE
자체 도메인 설정에 따른 팔로우 버튼 반응

### DISCUSSION THREAD

**Discussion by laterre39** [pronist/hELLO#117] [Q&A] [answered]
안녕하세요 이번에 블로그 시작하면서 자체 도메인 설정을 하게 되었는데

기본 도메인이 아닌 자체 도메인을 사용하는 경우에 팔로우 버튼이 작동되지 않고 그밖에 기타

로그인 관련 기능들도 작동하지 않습니다

할라면 코드를 수정해서 리다이렉트를 해야할듯 한데..

그밖에 방법이 있는지 궁금해서 개발자님의 의견을 듣고 싶어서 찾아왔습니다

언제나 이쁜 스킨 감사합니다.

**Comment by pronist** [comment, marked as answer]
@INewWorldI 안녕하세요!

2차 도메인의 로그인 및 댓글 등에 대한 제한은 티스토리 개발팀에서 HTTPS 보안 제약으로 막아 놓았습니다. 티스토리 개발팀에서도 이 부분 만큼은 고심한 것으로 보이는데 그렇다고 HTTPS 를 적용 안 할 수는 없는 노릇이라 희생한 것으로 보입니다.  특히 애초에 로그인 자체가 안 된 것으로 인지 되는지라 해결하기 어려울 수 있습니다. 

억지로 리다이렉트 한다고 해도 팔로우, 댓글, 좋아요 등을 누르려면 사용자가 블로그의 원 주소로 이동해야 하는 UX 상의 문제가 있을 것입니다. 사실 저 같아도 팔로우 버튼 눌렀는데 리다이렉트 되고 한 번 더 누른다는 생각은 전혀 안 할 것 같기도 하고 그러고 싶지도 않아질 것 같긴 합니다. 

저도 2차 도메인을 사용하면서 브랜딩 목적으로 그에 대한 기능은 희생하였기는 한데, 편법으로 하기엔 사용자 경험이 안 좋고 본질적으로는 해결 방법이 스킨 개발자 입장에서는 없는 것으로 보입니다. 그 밖에, 아예 2차 도메인 접속 시 자동으로 원 블로그로 리다이렉트한다?... 는 기능은 복구시킬 수 있는데, 2차 도메인 쓰는 의미가 없어지고 말이죠 ㅎ

**Reply by laterre39** [reply]
개발자님 상세한 답변 정말 감사드립니다. 
아쉬운 상황이기는 한데 개발자 팀의 고충도 충분히 이해가 가는 부분이네요
상황을 봐서 두 개의 도메인을 같이 용도에 맞추어서 사용을 하던지 해야겠습니다.
감사합니다!
